packages/wasmkit/src/lib/deploy/contract.ts

```diff
@@ -264,30 +264,36 @@ export class Contract {
     return instantiateInfo;
   }
 
-  async queryMsg (msgData: Record<string, unknown>): Promise<any> { // eslint-disable-line  @typescript-eslint/no-explicit-any
+  async queryMsg (msgData: Record<string, unknown>, skipLogs?: boolean): Promise<any> { // eslint-disable-line  @typescript-eslint/no-explicit-any
```

Comment:
Set a default value here as false, or handle if only this thing is true, then print the logs